site/scripts/generateHeaders.js

```diff
@@ -0,0 +1,80 @@
+const { writeFile } = require('fs/promises')
+const path = require('path')
+
+const fontsSrc = ['fonts.gstatic.com']
+
+const styleSrc = [
+  'fonts.googleapis.com',
+  `'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='`,
+  `'sha256-PlRfgfltn3VMXM8qgB4ESajyldRqslkTW9V28Xe7rQ0='`
```

Comment:
Where these two values come from? A comment should state that and how to retrieve/compute those again in case something changes.

Reply:
comment added

Reply:
> and can be obtained
> by running the application with [npm run serve] and
> checking the error messages in the web browser console

Hey, does this mean that on every build, these values will be different? As every change will impact the build, it seems that it will be easier for us to forget to update this the next time we run a change 🤔

Reply:
@gndelia I really don't understand how exactly this sha256 are generated, because it behaves different on each project. In this case, on Pure Finance project, the hash doesn't change on every build and when I make some changes in the CSS files, it always stays the same, I believe it represents the Tailwind CSS files (that's why it doesn't change).

So when there is no change on the sha256 hash between different builds and code changes I add it to the CSP header, but when the hash changes in build I don't add it to the header and seek an alternative.

Reply:
We should add a comment to each hash saying what exact script relates to. In addition, those hashes must be checked to be valid. Otherwise the build process should fail.

Reply:
@alvarengathomas please add the comment/reference to Tailwind in the comment in the code. It would be much easier to read it from there. A link to your comment above would work as well.